zhaquirks/tuya/ts011f_plug.py

```diff
@@ -35,6 +35,16 @@
 from zhaquirks.tuya.mcu import EnchantedDevice
 
 
+# Tuya Zigbee Metering Cluster Correction Implementation
+class TuyaZBMeteringCluster_v2(CustomCluster, Metering):
+    """Divides the kWh for tuya."""
+
+    UNIT_OF_MEASURE = 0x0300
+    MULTIPLIER = 0x0301
+    DIVISOR = 0x0302
+    _CONSTANT_ATTRIBUTES = {UNIT_OF_MEASURE: 0, MULTIPLIER: 1, DIVISOR: 100}
```

Comment:
I think it's best to move this v2 cluster class below the existing ``TuyaZBMeteringCluster`` in Tuya's ``__init__.py`` here:
https://github.com/zigpy/zha-device-handlers/blob/22968954cacaff4fc2b00e02ca1bd5622f637393/zhaquirks/tuya/__init__.py#L989-L995

--- 

If I'm not mistaking, we could probably add the "v2" cluster like following there, but it's also not really necessary (just prevents the need to re-define the attribute ids of ``MULTIPLIER, DIVISOR`` (for which we could probably also reference ``zigpy``)) -- but no need to:

```python
class TuyaZBMeteringCluster_v2(TuyaZBMeteringCluster):
    """Divides the kWh for Tuya and adds measurement unit."""

    UNIT_OF_MEASURE = 0x0300
    _CONSTANT_ATTRIBUTES = {UNIT_OF_MEASURE: 0, MULTIPLIER: 1, DIVISOR: 100}
```

---

And maybe there's a better name than "v2" for the cluster. I can't think of a good one either really, but how about ``TuyaZBMeteringClusterWithUnit``?

Reply:
Or let the PR being merged and after move the function to INIT as one "housekeeping PR".

Reply:
With local quirks, I don't seem to be able to override the __init__.py locally?

Reply:
I normally doing the INIT changes in the container but you can doing locally but need changing the import path for getting it working.

Testing your quirk is working OK with the changes you have made and then you can testing putting it in the INIT in the container and see if you is getting it  working.
Instruction for editing quirks  in the HA container https://github.com/zigpy/zha-device-handlers/discussions/693.

Reply:
Thanks, I've done it "by eye" and the tests pass, so assume it's ok. Will use the container testing if I have to do more complex stuff in future, thanks

Reply:
If having good eye its normally being OK then the tests is being OK.

### Great work done !!!